### PR TITLE
[homework #2] - HTML by Edelmiro Anton

### PR DESCRIPTION
**Pull request by EdelmiroAnton**
@ilya-ischenko Hi Ilya! Let me know if i need to do any change.

### CODE REVIEW

**Review by ilya-ischenko — COMMENTED** [review]
That's ok, but it would be good to make some changes:

1. Let's try to make file management, move images and icons in the separate folder/s. It will be more convenient to navigate in your project. It is a really important thing in the bigger projects. 

2. Did you use prettier? It seems that a lot of thing was not formatted in the good way. Write in slack if you have some issues with it.

3. Let's try to do your pages more semantically correct. Try to use "section" element for example.

File: homeworks/edelmiro-anton/02-html/contact.html [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+            </label>
+            <label>
+                <input type="text"  placeholder="Your last name..." required>   
+            </label>
+        </div>
+
+        <div>
+            <label>
+                Phone number*
+                <input type="number"  placeholder="Your phone..." required>
```
Comment (ilya-ischenko) [inline homeworks/edelmiro-anton/02-html/contact.html:35]: We have type="tel" value for this field

File: homeworks/edelmiro-anton/02-html/home.html [hunk 2]
```diff
@@ -31,7 +31,6 @@ <h1>Fron-end Developer </h1>
 
             <img src="./img/edel.png" alt="a photo of myself" width="250">
         </section>
```
Comment (EdelmiroAnton) [inline homeworks/edelmiro-anton/02-html/home.html:33]: Section tag was added in previous commit.

File: homeworks/edelmiro-anton/02-html/home.html [hunk 3]
```diff
@@ -31,7 +31,6 @@ <h1>Fron-end Developer </h1>
...
-
```
Comment (EdelmiroAnton) [inline homeworks/edelmiro-anton/02-html/home.html]: Section tag was added in previous commit.

File: homeworks/edelmiro-anton/02-html/contact.html [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+            </label>
+        </div>
```
[the remark this answers is not in the document]
Comment (EdelmiroAnton) [reply homeworks/edelmiro-anton/02-html/contact.html]: done!

File: homeworks/edelmiro-anton/02-html/contact.html [hunk 5]
```diff
@@ -0,0 +1,309 @@
...
+
+        Country
+        <select name="Country">
```
[the remark this answers is not in the document]
Comment (EdelmiroAnton) [reply homeworks/edelmiro-anton/02-html/contact.html]: done!

File: homeworks/edelmiro-anton/02-html/contact.html (hunk 1 above)
Comment (EdelmiroAnton) [reply homeworks/edelmiro-anton/02-html/contact.html]: done!

**Review by ilya-ischenko — APPROVED** [review]
Well done!

**Pull request merged**